### DISCUSSION TITLE
Cover page for PDF Book

### DISCUSSION THREAD

**Discussion by jpmorr** [quarto-dev/quarto-cli#1941] [Q&A] [answered]
I'm wondering how you go about making a cover page that is more complete than the default '_bare-bones'_ version.

When I've written report/manuals/etc in `LaTeX`, it's very easy to have a title page that has title, author(s), affiliation(s), some sort of image and/or possibly a organisation logo on the cover page. If there's no `verso` page the cover may also include details such a copyright info, document version, doi or publishing date towards the bottom.

Is there anyway to create such _feature_ filled cover pages and/or verso pages with `quarto`.

I'm really enjoying the transition to quarto, although I'm struggling to figure out how to do some things like editing formatting for sections headers or captions. Are there any examples that show some of the more advanced customisations that can be done for creating pdfs?

**Comment by jjallaire** [comment, marked as answer]
Yes, definitely! Check this out: https://nmfs-opensci.github.io/quarto_titlepages/

**Reply by jpmorr** [reply]
Wonderful!

**Comment by fuhrmanator** [comment]
The titlepages template/extension is very cool. However, if you want a simple way to include an image on the cover page of a `scrbook` class before the title, you can use the `titlehead` hook in the LaTeX headers:

```yaml
format:
  pdf:
    header-includes: |
      \titlehead{\includegraphics[width=6in]{cover-image.png}}
```

I didn't try it, but I the [Koma Script classes](https://mirror.csclub.uwaterloo.ca/CTAN/macros/latex/contrib/koma-script/doc/scrguide-en.pdf) will also display a `\publishers` part _after_ the date, before the `\thanks` in a footnote at the bottom.

**Reply by zawkzaw** [reply]
Hello!
Thank you for a simple solution for including  an image on the cover page of quarto pdf book.  I tried this and log was there  but it is upper left corner. If I want it on other places, say upper  center place or lower centre place.

Thanks in advance

**Reply by fuhrmanator** [reply]
Hello -- this [answer](https://tex.stackexchange.com/a/210280/17868) should help you a lot. You can also ask ChatGPT (or Copilot) to help you customize the title page in LaTeX, but you need to work the LaTeX solution into Quarto with the `header-includes` as above.

**Comment by friendly** [comment]
Does the simple `header-includes` solution also work for other document classes? I'm using `krantz.cls` for my book

**Reply by Steinthal** [reply]
I guess you're still using `include-in-header` (according to #11087), so you could add the `titlehead` instruction in your `latex/preamble.tex`file.

**Reply by friendly** [reply]
`\titlehead` is defined somewhere in what you get from `scrbook.cls` (Koma script package), but I haven't been able to find where/how it is defined to be able to use that in my preamble. 

In my case I have a **complete** `.png` image of my cover, with title, author, ... and a graphic image, so nothing else is needed.
I don't think I can use the `quarto_titlepages` extension for this, so I need to define something in LaTeX that will essentially use `\includegraphics[width=6in]{cover-image.png}` on a separate page.
